.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,19 @@
+<!--
+
+Template: QUESTION
+
+This is an issue for a feature request.
+You should label `question`.
+
+
+If your purpose is different, please visit following links.
```

Comment:
Suggestion: "If you issue is not a question, please visit one of the following links."

Reply:
👍